sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/login/AADLoginIT.java

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.login;
+
+import com.azure.test.utils.AppRunner;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.oauth2.client.oidc.userinfo.OidcUserRequest;
+import org.springframework.security.oauth2.client.userinfo.OAuth2UserService;
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.function.Consumer;
+import java.util.regex.Pattern;
+
+import static com.azure.test.oauth.OAuthUtils.AAD_MULTI_TENANT_CLIENT_ID;
+import static com.azure.test.oauth.OAuthUtils.AAD_MULTI_TENANT_CLIENT_SECRET;
+import static com.azure.test.oauth.OAuthUtils.AAD_TENANT_ID_1;
+import static com.azure.test.oauth.OAuthUtils.AAD_USER_NAME_1;
+import static com.azure.test.oauth.OAuthUtils.AAD_USER_PASSWORD_1;
+import static org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated;
+
+public class AADLoginIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AADLoginIT.class);
+
+    static {
+        final String chromedriverLinux = "src/test/resources/driver/chromedriver_linux64";
+        final String chromedriverWin32 = "src/test/resources/driver/chromedriver_win32.exe";
+        final String chromedriverMac = "src/test/resources/driver/chromedriver_mac64";
+        String osName = System.getProperty("os.name").toLowerCase();
+        if (Pattern.matches("linux.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverLinux);
+        } else if (Pattern.matches("windows.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverWin32);
+        } else if (Pattern.matches("mac.*", osName)) {
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, chromedriverMac);
+        }
+    }
+
+//    @Test
```

Comment:
When can we enable it?

Reply:
When you delete the application.properties

Reply:
Already deleted in latest master branch. Please merge latest master branch to your branch. 🙏